.github/workflows/fedora-copr-build.yml

```
@@ -13,28 +13,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        name: [standalone, big-merge, bootstrap]
+        name: [big-merge, bootstrap]
         include:
-          - name: standalone
-            copr_project_tpl: "@fedora-llvm-team/llvm-snapshots-incubator-YYYYMMDD"
-            copr_target_project: "@fedora-llvm-team/llvm-snapshots"
-            clone_url_tpl: "https://src.fedoraproject.org/rpms/PKG.git"
-            clone_ref: upstream-snapshot
-            needs_llvm_snapshot_builder: true
+          # - name: standalone
+          #   copr_project_tpl: "@fedora-llvm-team/llvm-snapshots-incubator-YYYYMMDD"
+          #   copr_target_project: "@fedora-llvm-team/llvm-snapshots"
+          #   clone_url_tpl: "https://src.fedoraproject.org/rpms/PKG.git"
+          #   clone_ref: upstream-snapshot
           - name: big-merge
             copr_project_tpl: "@fedora-llvm-team/llvm-snapshots-big-merge-YYYYMMDD"
-            copr_target_project: "@fedora-llvm-team/llvm-snapshots-big-merge"
+            copr_target_project: "@fedora-llvm-team/llvm-snapshots"
             extra_script_file: "scripts/functions-big-merge.sh"
             clone_url_tpl: "https://src.fedoraproject.org/forks/kkleine/rpms/PKG.git"
```

```suggestion
            clone_url_tpl: "https://src.fedoraproject.org/rpms/PKG.git"
```
I think this should point to the main repo now?

---

That is correct. Let me fix it.